packages/core/core/src/Parcel.js

```diff
@@ -200,6 +200,7 @@ export default class Parcel {
       this.#assetGraphBuilder.writeToCache(),
       this.#runtimesAssetGraphBuilder.writeToCache(),
     ]);
+    this.#reporterRunner.destroy();
```

Comment:
nit: we've been using `.dispose()` as the name for methods that free resources. The `Disposable` class can also manage disposing of many objects that implement that interface.